src/python/engagement_edge/src/engagement_edge.py

```diff
@@ -418,17 +160,18 @@ def get_salt_and_pw(table, username):
         LOGGER.debug(f"Did not get salt for user: {username}")
         return None, None
 
-    salt = response["Item"]["salt"].value
-    password = response["Item"]["password"]
+    # Not quite sure what type this is supposed to be.
+    salt = response["Item"]["salt"].value  # type: ignore
```

Comment:
basically, mypy doesn't think there's anything that could be in here that responds to `.value`